src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/tests/Popover.test.js

```diff
@@ -54,13 +50,16 @@ test('The popover should render in body when open', () => {
             }
         </Popover>
     );
-    expect(view.find('Backdrop')).toHaveLength(1);
-    expect(view.find('Backdrop').prop('open')).toEqual(true);
-    expect(pretty(body.innerHTML)).toMatchSnapshot();
+
+    popover.instance().popoverWidth = 20;
+    popover.instance().popoverHeight = 100;
+    popover.update();
+
+    expect(popover.instance().dimensions.scrollTop).toBe(4);
```

Comment:
should i understand this? 🙈

Reply:
I don't understand it completely as well :see_no_evil: But this is a check if the positioning is working, which does not seem to reflected in the snapshot, so I included it here as well. Didn't want to delete too much :see_no_evil: